pkg/apis/flows/v1/sequence_lifecycle_test.go

```diff
@@ -45,7 +45,7 @@ func TestSequenceGetConditionSet(t *testing.T) {
 func getSubscription(name string, ready bool) *messagingv1.Subscription {
 	s := messagingv1.Subscription{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "eventing.knative.dev/v1alpha1",
+			APIVersion: "eventing.knative.dev/v1",
```

Comment:
subscription s in `messaging`, see: https://github.com/knative/eventing/blob/main/config/core/resources/subscription.yaml#L17